homeassistant/components/xiaomi_miio/fan.py

```diff
@@ -227,6 +240,28 @@
     ATTR_ILLUMINANCE: "illuminance",
 }
 
+AVAILABLE_ATTRIBUTES_AIRPURIFIER_3 = {
+    ATTR_TEMPERATURE: "temperature",
+    ATTR_HUMIDITY: "humidity",
+    ATTR_AIR_QUALITY_INDEX: "aqi",
+    ATTR_MODE: "mode",
+    ATTR_FILTER_HOURS_USED: "filter_hours_used",
```

Comment:
So, should this be completely removed then? The problem with this value is, I presume, that there is no specific timestamp as it depends on the usage.

P.S. I think we need to have a checklist for reviewers, much like what we have for PR creators, to list things to check. It is sometimes hard to spot all mistakes, and I'm very sorry for not catching this one..

Reply:
We should update this dev docs page with the allowed fan speeds:
https://developers.home-assistant.io/docs/entity_fan

I'll put it on my todo.

In general we need to make sure that we follow our architecture docs and ADRs. But those are not complete yet, so that's a problem.

Reply:
It's interesting that relative timestamps are not allowed in state attributes. Didn't know that. Where can I find more docs about it?
I have MQTT devices, Esphome which are breaking this rule. This cannot be found in HA code as HA just puts received json in `state_attrs`.
my vacuum state attributes:
```
cleanTime: 202.8
cleanArea: 12530.2
cleanCount: 1349
last_run_stats: 
  startTime: 1585934330000
  endTime: 1585935335000
duration: 1005
```

@rytilahti solution for it is to move attributes to independent sensors.